src/components/AccordionButton/AccordionButton.module.css

```
@@ -0,0 +1,49 @@
+/*------------------------------------*\
+    # ACCORDION BUTTON
+\*------------------------------------*/
+
+/**
+ * Accordion Button, wraps the heading and open indicator icon.
+ */
+.accordion-button.accordion-button {
+  display: flex;
+  justify-content: space-between;
+  padding: var(--eds-size-2) var(--eds-size-1);
+  height: 3.375rem;
+}
+
+/**
+ * Compact variant.
+ */
+.accordion-button--compact.accordion-button--compact {
+  padding: var(--eds-size-1);
+  height: 2.25rem;
+}
+
+/**
+ * Expand more (chevron) icon indicates open or closed status.
+ *
+ * This nonrotated icon points down and represents closed status.
+ */
+.accordion-button__icon {
+  transform: rotate(0);
+}
+/**
+ * This rotated icon points up and represents open status.
+ */
+.accordion-button__icon--open {
+  transform: rotate(180deg);
+}
+
+/**
+ * Animates the icon rotation when opening and closing.
+ */
+.accordion-button > .accordion-button__icon {
+  transition-property: transform;
+  transition-duration: var(--eds-anim-move-medium);
+  transition-timing-function: var(--eds-anim-ease);
```

Just curious, is there a particular reason you split this across 3 lines? Is it for readability?

```suggestion
  transition: transform var(--eds-anim-move-medium) var(--eds-anim-ease);
```

---

Also, we can't this stuff go in `.accordion-button__icon {`? Is it a specificity issue?

---

Nah I like the condensed, thanks for the suggestion.

And yea it's more specificity issues https://github.com/chanzuckerberg/edu-design-system/blob/e519e4c6cfc0f849354142c2967a261c23a78b4c/src/components/ClickableStyle/ClickableStyle.module.css#L23-L25